pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,26 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "venus_relief": GMTRemoteDataset(
+        title="Venus relief",
+        name="venus relief",
+        long_name="Venus relief",
```

Comment:
Please see https://www.generic-mapping-tools.org/remote-datasets/venus-relief.html.

```suggestion
        title="Venus relief",
        name="venus relief",
        long_name="NASA Magellan Venus relief",
```